test/SlowTests/Issues/RavenDB_21233.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Documents.Queries;
+using Tests.Infrastructure;
+using Xunit;
+
+
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_21233 : RavenTestBase
+    {
+        private IndexQuery q1 = new()
+        {
+            Query = @"
+from ""Orders"" update {
+    for (var i = 0; i < 10000; i++){
+        this.Freight = 13;
+    }
+}"
+        };
+
+        private IndexQuery q2 = new()
+        {
+            Query = @"
+from index ""Orders/ByCompany""
+ update {
+    for (var i = 0; i < 10000; i++){
```

Comment:
change the default db setting to 100 and adjust this value accordingly, if not this test can run for a long time unnecessarily

Reply:
Done